src/cromshell/utilities/io_utils.py

```diff
@@ -1,11 +1,13 @@
 import json
 import logging
 import re
+import os
```

Comment:
If possible use avoid using os package and use pathlib.Path library for file/directly handling. The pathlib module is a newer module in Python that provides a more object-oriented way of handling file paths. This makes it more intuitive and easier to use, and it also provides a number of features that are not available in the os module.